ihatemoney/templates/invitation_mail.fr.j2

```diff
@@ -1,11 +1,11 @@
 Salut,
 
-Quelqu'un avec l'adresse "{{ g.project.contact_email }}" vous à invité à partager vos dépenses pour "{{ g.project.name }}".
+Quelqu'un dont l'adresse email est {{ g.project.contact_email }} vous a invité à partager vos dépenses pour « {{ g.project.name }} ».
 
-C'est aussi simple que de dire qui à payé pour quoi, pour qui, et combien celà à coûté, on s’occupe du reste.
+Il suffit de renseigner qui a payé pour quoi, pour qui, combien ça a coûté, et on s’occupe du reste.
 
-Vous pouvez vous authentifier avec le lien suivant: {{ url_for(".authenticate", _external=True, token=g.project.generate_token()) }}.
-Une fois authentifié, vous pouvez utiliser le lien suivant qui est plus facile à mémoriser: {{ url_for(".list_bills", _external=True) }}
-Si votre cookie est supprimé ou si vous vous déconnectez, vous devrez vous authentifier à nouveau en utilisant le premier lien.
+Vous pouvez vous connecter grâce à ce lien : {{ url_for(".authenticate", _external=True, token=g.project.generate_token()) }}.
+Une fois connecté, vous pourrez utiliser le lien suivant qui est plus facile à mémoriser : {{ url_for(".list_bills", _external=True) }}
+Si votre cookie est supprimé ou si vous vous déconnectez, vous devrez vous connecter à nouveau en utilisant le premier lien.
```

Comment:
I wonder if we need to specify this abotu the cookie stuff, most people don't understand what this means.

Reply:
```suggestion
Si vous êtes déconnecté volontairement ou non, vous devrez utiliser à nouveau le premier lien.
```
It removes the cookie mention, and simplify the last part, in order to avoid redundancy. What do you think?